locales/locales/en.json

```diff
@@ -2043,6 +2043,12 @@
 	"betteRedditRestrictScrollEventsDesc": {
 		"message": "Reddit performs much work every time you scroll, which may cause lag on pages with many comments. Reducing the number of scroll events it processes improves performance."
 	},
+	"betteRedditunEscapeLinksDesc": {
+		"message": "Fix Escaped Links"
+	},
+	"betteRedditunEscapeLinksTitle": {
+		"message": "New Reddit adds backslashes before underscores and quotes in links."
```

Comment:
I would make this much more specific (i.e. "New Reddit adds backslashes before underscores and quotes in links, which often causes links to be broken.")

Reply:
Yeah I had no idea how to word this, feel free to change it.